paymentsheet/build.gradle

```diff
@@ -7,7 +7,7 @@ apply plugin: 'app.cash.paparazzi'
 
 dependencies {
     implementation project(":link")
-    implementation project(":payments-core")
+    api project(":payments-core")
```

Comment:
With `PaymentMethod` now being part of `paymentsheet`’s API, this should be an `api` dependency.